modules/data/schematics/ng-add/__snapshots__/index.spec.ts.snap

```diff
@@ -0,0 +1,33 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Data ng-add Schematic Migration of ngrx-data Data ng-add Schematic for standalone application provides data without effects 1`] = `
+"import { ApplicationConfig } from '@angular/core';
+import { provideEntityData } from '@ngrx/data';
+import { entityConfig } from './entity-metadata';
+
+export const appConfig: ApplicationConfig = {
+  providers: [provideEntityData(entityConfig)]
+};
+"
+`;
+
+exports[`Data ng-add Schematic Migration of ngrx-data Data ng-add Schematic for standalone application provides data without entityConfig 1`] = `
+"import { ApplicationConfig } from '@angular/core';
+import { provideEntityData, withEffects } from '@ngrx/data';
+
+export const appConfig: ApplicationConfig = {
+  providers: [provideEntityData(withEffects())]
```

Comment:
The `provideEntityData` function expects `entityConfig` as the required argument. Check the function signature here: https://github.com/ngrx/platform/blob/main/modules/data/src/provide-entity-data.ts#L240

If the `entityConfig` option is `false`, we can pass an empty object:

```ts
  providers: [provideEntityData({}, withEffects())],
```